pkgs/by-name/ew/eww/package.nix

```diff
@@ -21,9 +23,22 @@ rustPlatform.buildRustPackage rec {
     hash = "sha256-rzDnplFJNiHe+XbxbhZMEhPJMiJsmdVqtZxlxhzzpTk=";
   };
 
+  patches = [
+    # FIXME: remove with the next release
+    # https://github.com/elkowar/eww/pull/1084 - patch to fix string truncation in eww config
+    (fetchpatch {
+      name = "fix-eww-string-truncation.patch";
+      url = "https://github.com/elkowar/eww/commit/2c8811512460ce6cc75e021d8d081813647699dc.patch";
+      hash = "sha256-oYrpuTI8KbMtboOdc8xw3UrAEkLu5xnphgey7koKmrc=";
+    })
+  ];
```

Comment:
Is there a reason for not just changing the source ref to the commit you're fetching here?

Reply:
I thought the same, since it also [incorporates](https://github.com/elkowar/eww/compare/v0.6.0...2c8811512460ce6cc75e021d8d081813647699dc) Cargo.toml and other changes I was hesitant.
Second opinion, @eclairevoyant?

Reply:
I'd be really surprised if those changes cause issues

Reply:
Yep, thought the same :) looked at the changes, pretty harmless bugfixes and deps update

Reply:
Either is fine, though if you're changing the commit, you'll also have to change the version to the unstable format.

Reply:
Using the version `0.6.0-unstable-2024-04-26`.